### PR TITLE
[#176013578] added code-review and deploy pipeline

### PR DESCRIPTION
**Pull request by gquadrati**
- added code-review pipeline (danger.js check only)
- added deploy pipeline (node version 10.18.0)
- added yarn.lock file
- added auto-changelog and related scripts and files
- TODO:
  [x] create pipeline code-review
  [x] create deploy pipeline

### CODE REVIEW

File: .devops/deploy-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+              # On the assumption that this stage is executed only when Relase stage is,
+              #  with this parameter we set the reference the deploy script must pull changes from.
+              # The branch/tag name is calculated from the source branch
+              #  ex: Build.SourceBranch=refs/heads/master --> master
+              #  ex: Build.SourceBranch=refs/tags/v1.2.3-RELEASE --> v1.2.3-RELEASE
+              gitReference: ${ gitReference }
+
+          - script: |
+              yarn pack
+            displayName: 'Pack'
```
Comment (balanza) [inline .devops/deploy-pipelines.yml:96]: why do we need this script?

Comment (gquadrati) [reply .devops/deploy-pipelines.yml:96]: I thought it was necessary to create the package

Comment (balanza) [reply .devops/deploy-pipelines.yml:96]: i's actually done internally by the `yarn publish` command, which is itself executed by the Npm azure task

Comment (balanza) [reply .devops/deploy-pipelines.yml:96]: https://docs.npmjs.com/cli/v6/commands/npm-pack

Comment (gquadrati) [reply .devops/deploy-pipelines.yml:96]: ok, sorry I didn't know.. so I suppose I need to add .ios and android folders to the file list, to let them be packed, right?

Comment (balanza) [reply .devops/deploy-pipelines.yml]: Good point. How was this package published so far? We should mimic such process cc @Undermaken

**Review by balanza — APPROVED** [review]

**Pull request merged**